### PR TITLE
Add retraining time

### PR DESCRIPTION
**Pull request by StanislasGuinel**

### CODE REVIEW

File: python-lib/gluonts_forecasts/model.py [hunk 1]
```diff
@@ -163,7 +175,7 @@ def _format_metrics(self, agg_metrics, item_metrics, train_list_dataset):
 
         item_metrics[METRICS_DATASET.TARGET_COLUMN] = target_columns
         agg_metrics[METRICS_DATASET.TARGET_COLUMN] = METRICS_DATASET.AGGREGATED_ROW
-        agg_metrics[METRICS_DATASET.TRAINING_TIME] = round(self.evaluation_time, 2)
+        agg_metrics[METRICS_DATASET.TRAINING_TIME] = round(self.evaluation_time + self.retraining_time, 2)
```
Comment (alexcombessie) [inline python-lib/gluonts_forecasts/model.py:178]: After thinking again about this, not sure we should round. If the user choose only basic model like seasonal_naive, rounding would mean 0.
```suggestion
        agg_metrics[METRICS_DATASET.TRAINING_TIME] = self.evaluation_time + self.retraining_time
```

**Review by alexcombessie — APPROVED** [review]
LGTM! One very small nitpick, otherwise it is good to merge!

File: python-lib/gluonts_forecasts/model.py (hunk 1 above)
Comment (StanislasGuinel) [reply python-lib/gluonts_forecasts/model.py:178]: round with 4 decimals maybe ?
having the complete float number is not so friendly

Comment (alexcombessie) [reply python-lib/gluonts_forecasts/model.py:178]: I'd say to leave it without rounding, since it's hard to know in advance how precise users want to be

Comment (alexcombessie) [reply python-lib/gluonts_forecasts/model.py:178]: I'd suppose the types of users interested in this are more coders, so they'd value precision. The analyst users would probably never use that column.

Comment (StanislasGuinel) [reply python-lib/gluonts_forecasts/model.py]: done in 71bb631

**Review by alexcombessie — APPROVED** [review]

**Pull request merged**